src/render/PanelHeaderProvider.js

```diff
@@ -185,7 +185,7 @@ function getTemplatesService() {
 
 function getTemplate(element, elementTemplates) {
   const templateId = elementTemplates._getTemplateId(element);
-  return templateId && elementTemplates.get(templateId);
+  return templateId && elementTemplates.get(element);
```

Comment:
```suggestion
  return elementTemplates.get(element);
```

Reply:
That's true as we anyway check for `template && ...` anywhere we use this helper.